### PR TITLE
feat: simple user creation form

### PR DESCRIPTION
**Pull request by Salzig**
feat: simple user creation form

first steps for #283 and #179

### CODE REVIEW

File: app/views/admin/users/new.html.slim [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    = form_for [:admin, @user], html: {class: 'form-horizontal', role: 'form'} do |f|
+      .form-group
+        = f.label :username, {class: 'control-label col-md-2'}
+        .col-md-7
+          = f.text_field(:username, class: 'form-control', required: true, autofocus: true)
+      .form-group
+        = f.label :email, {class: 'control-label col-md-2'}
+        .col-md-7
+          = f.text_field(:email, class: 'form-control', required: true)
```
Comment (mssola) [inline app/views/admin/users/new.html.slim]: use `f.email_field` instead.

**Comment by mssola** [comment]
Awesome :) !

I have one issue though. When the admin makes a mistake, it gets displayed like this:

form

In order to get it properly, you have to change in the line 20 of the `app/controllers/admin/users_controller.rb` file:

```ruby
render "new"
```

to:

```ruby
edirect_to new_admin_user_path, alert: @user.errors.full_messages
```

Then it will look like this:

form2

Besides that, everything looks ok to me. I'll update the related issue with a checklist of the steps to get this feature implemented. Thanks for stepping in!

File: app/views/admin/users/new.html.slim (hunk 1 above)
Comment (Salzig) [inline app/views/admin/users/new.html.slim]: good catch, will change that later.

**Comment by Salzig** [comment]
bildschirmfoto 2015-11-09 um 23 46 19

**Comment by mssola** [comment]
LGTM. Could you squash it into a single commit?

**Comment by Salzig** [comment]
Sure, will do that later.

**Comment by Salzig** [comment]
done.

**Comment by mssola** [comment]
@Salzig thanks!

**Pull request merged**